products/java.md

```diff
@@ -12,16 +12,21 @@ releasePolicyLink: https://www.oracle.com/technetwork/java/java-se-support-roadm
 releaseDateColumn: true
 sortReleasesBy: 'releaseCycle'
 releases:
+-   releaseCycle: "19"
+    support: 2022-03-31
+    eol: 2022-03-31
```

Comment:
Since it's vague, shouldn't we use 2022-03-01 instead?

Reply:
Updated. I had the year wrong, it's 2023 :smile:

Reply:
@BiNZGi looks like you love 2022 :D it is easy to write, just wait 2222 it will be awesome :D ❤️